client/src/uxManager.js

```diff
@@ -268,5 +268,38 @@ class UxManager extends LuigiClientBase {
   getCurrentTheme() {
     return lifecycleManager.currentContext?.internal?.currentTheme;
   }
+
+  /**
+   * Gets the CSS variables from Luigi Core with their key and value.
+   * @returns {Object} CSS variables with their key and value.
+   * @memberof uxManager
+   * @since NEXTRELEASE
+   * @example LuigiClient.uxManager().getCSSVariables();
+   */
+  getCSSVariables() {
+    return lifecycleManager.currentContext?.internal?.cssVariables;
```

Comment:
```suggestion
    return lifecycleManager.currentContext?.internal?.cssVariables || {};
```
otherwise it should be documented that return value can be undefined